javacord-api/src/main/java/org/javacord/api/entity/message/Message.java

```diff
@@ -719,10 +721,20 @@ default CompletableFuture<Void> unpin() {
         return Message.unpin(getApi(), getChannel().getId(), getId());
     }
 
+    /**
+     * Checks if the bot can read the content of the message.
+     *
+     * @return Whether the bot can read the content of the message.
+     */
+    boolean canYouReadContent();
+
     /**
      * Gets the content of the message.
      *
      * @return The content of the message.
+     * @throws MissingIntentException If you are missing the {@link Intent#MESSAGE_CONTENT} intent which is needed to
+     *                                receive the content of a message except if the message
+     *                                is a DM, you are mentioned, or it's your own message.
```

Comment:
```suggestion
     * @throws MissingIntentException If you can't get the message content because you are not mentioned in the
     *                                Message, not the author of the message, the message is not a DM, and you
     *                                don't have the {@link Intent#MessageContent} intent set.
```
Existing description reads like an exception is always thrown when the intent is missing.